lib/element.rb

```diff
@@ -202,16 +238,6 @@ def text
     element.text
   end
 
-  def text=(text)
```

Comment:
This will certainly break NLVX tests that use this method to set text values. Additionally, since this is a publicly used gem, this may break other tests things we don't even know about.